pallets/gear-built-in-actors/common/src/lib.rs

```diff
@@ -0,0 +1,116 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2023 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Helper crate defining necessary types for messages to Gear built-in actor.
+
+#![no_std]
+
+extern crate alloc;
+
+use alloc::vec::Vec;
+use scale_info::scale::{self, Decode, Encode};
+
+// /// Actor id as it is seen by a contract.
+// #[derive(Encode, Decode, Clone, Copy, Default, PartialEq, Eq, Debug)]
+// #[codec(crate = scale)]
+// pub struct ActorId(pub [u8; 32]);
+pub type AccountId = [u8; 32];
```

Comment:
We don't want the `gstd` as a dependency to avoid cyclic dependencies issue.
It's a type alias only after all - not even a new type. Will remove the dead code not to cause confusion.